server/lib/tuist_web/live/members_live.ex

```diff
@@ -259,6 +422,44 @@ defmodule TuistWeb.MembersLive do
     {:noreply, socket}
   end
 
+  def handle_event("select-member-role", %{"member_id" => member_id, "role" => role}, socket) do
+    member_id_int = String.to_integer(member_id)
+    socket = assign(socket, managing_member: {member_id_int, role})
+    {:noreply, socket}
+  end
+
+  def handle_event("save-member-role", %{"member-id" => member_id}, socket) do
+    member_id_int = String.to_integer(member_id)
+    {^member_id_int, new_role} = socket.assigns.managing_member
+
+    [member, _role] = Enum.find(socket.assigns.members, fn [m, _role] -> m.id == member_id_int end)
+    organization = socket.assigns.organization
+
+    Accounts.update_user_role_in_organization(member, organization, String.to_existing_atom(new_role))
```

Comment:
I'd match this against `:ok` to ensure the role is updated or that we raise otherwise.